packages/components/src/DateTimePickers/DateTime/date-extraction.js

```diff
@@ -423,6 +423,25 @@ function extractPartsFromDateTime(datetime, options) {
 	};
 }
 
+/**
+ * extract date/time part from dateFormat
+ * @param {string} dateFormat
+ * @param {string} part
+ */
+function extractFormatByPart({ dateFormat, useTime, useSeconds }, part = 'date') {
+	const formatParts = dateFormat
+		.split(/\s/g) // split from blank
+		.filter(formatPart => formatPart); // remove empty elements
+
+	if (part === 'date') {
+		return formatParts[0] || '';
+	}
+	if (useTime) {
+		return useSeconds ? 'HH:mm:ss' : 'HH:mm';
```

Comment:
It seems that in this new change, you only have date in « dateFormat ». So I don’t see the point in extracting the date part.

Let’s see the need again, you need to get the date format and time format for placeholders.
- date : just use the dateFormat from props
- time: you can have a getTimeFormat() that generate the format depending on useSeconds. 

No need of the extractFormatByPart anymore.

Reply:
Actually `dateFormat` can be `YYYY-MM-DD HH:mm` which includes time part.
So I still need this extractFormatByPart for get date part format.
maybe change the naming `dateFormat` to `datetimeFormat`?